docs/experiments-wiki-corpora.md

```diff
@@ -0,0 +1,286 @@
+# Pyserini: Reproducing DPR Results With Improved Wikipedia Corpus Variants
+
+Dense passage retriever (DPR) is a dense retrieval method described in the following paper:
+
+> Vladimir Karpukhin, Barlas Oğuz, Sewon Min, Patrick Lewis, Ledell Wu, Sergey Edunov, Danqi Chen, Wen-tau Yih. [Dense Passage Retrieval for Open-Domain Question Answering](https://www.aclweb.org/anthology/2020.emnlp-main.550/). _Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing (EMNLP)_, pages 6769-6781, 2020.
+
+We have replicated DPR results with our Wikipedia corpus variants.
+
+Our own efforts are described in the paper: 
+> Manveer Singh Tamber, Ronak Pradeep, and Jimmy Lin. "Pre-Processing Matters! Improved Wikipedia Corpora for Open-Domain Question Answering". ECIR 2023.
+
+This guide provides instructions to reproduce the pre-processing to generate the corpora and the retrieval on the ```wiki-all-6-3-tamber``` corpus variant.
+For end-to-end answer generation, please see [this guide](https://github.com/castorini/pygaggle/blob/master/docs/experiments-wiki-corpora-fid.md) in our PyGaggle neural text ranking and question answering library.
+
+## Generate Corpora
```

Comment:
Move this section all the way down to the version bottom, since in the next section you provide downloadable corpora already.